<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailsGroupBox</class>
 <widget class="QGroupBox" name="DetailsGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>317</height>
   </rect>
  </property>
  <property name="title">
   <string>File details options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>Print metadata fields:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="exifTab">
      <attribute name="title">
       <string>Exif</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="4" column="0">
        <widget class="QCheckBox" name="exifGeneratedTimeCheckBox">
         <property name="text">
          <string>Generated Date and Time</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="exifSoftCheckBox">
         <property name="text">
          <string>Processing Software</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="exifFocalLengthCheckBox">
         <property name="text">
          <string>Focal length</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="exifVersionCheckBox">
         <property name="text">
          <string>Exif Version</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="exifOrientationCheckBox">
         <property name="text">
          <string>Orientation</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="exifManufacturerCheckBox">
         <property name="text">
          <string>Camera Manufacturer</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="exifDigitizedTimeCheckBox">
         <property name="text">
          <string>Digitized Date and Time</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="exifApertureCheckBox">
         <property name="text">
          <string>Aperture</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="exifExpTimeCheckBox">
         <property name="text">
          <string>Exposure time</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QCheckBox" name="exifModelCheckBox">
         <property name="text">
          <string>Camera Model</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="exifIsoCheckBox">
         <property name="text">
          <string>ISO Speed</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QCheckBox" name="exifMeteringCheckBox">
         <property name="text">
          <string>Light metering mode</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QCheckBox" name="exifExpProgramCheckBox">
         <property name="text">
          <string>Exposure program</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QCheckBox" name="exifArtistCheckBox_D">
         <property name="text">
          <string>Artist</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QCheckBox" name="exifCopyrightCheckBox_D">
         <property name="text">
          <string>Copyright</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QCheckBox" name="exifUserCommentCheckBox_D">
         <property name="text">
          <string>User Comment</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QCheckBox" name="exifFlashCheckBox">
         <property name="text">
          <string>Flash mode</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="exifShutterSpeedCheckBox">
         <property name="text">
          <string>Shutter Speed</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="exifExpBiasCheckBox">
         <property name="text">
          <string>Exposure bias</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="iptcTab">
      <attribute name="title">
       <string>IPTC</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <widget class="QCheckBox" name="iptcBylineCheckBox">
         <property name="text">
          <string>Author</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="iptcVersionCheckBox">
         <property name="text">
          <string>Model version</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="iptcCopyrightCheckBox">
         <property name="text">
          <string>Copyright</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="iptcObjectNameCheckBox">
         <property name="text">
          <string>Object name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="iptcCreatedDateCheckBox">
         <property name="text">
          <string>Created date</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="iptcKeywordsCheckBox">
         <property name="text">
          <string>Keywords</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="iptcDigitizedDateCheckBox">
         <property name="text">
          <string>Digitized date</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="iptcCaptionCheckBox">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="iptcCountryNameCheckBox">
         <property name="text">
          <string>Country</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="iptcCityCheckBox">
         <property name="text">
          <string>City</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="iptcCreatedTimeCheckBox">
         <property name="text">
          <string>Created time</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="iptcDigitizedTimeCheckBox">
         <property name="text">
          <string>Digitized time</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QCheckBox" name="iptcEditStatusCheckBox">
         <property name="text">
          <string>Edit status</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
